assets/js/atomic/blocks/product/rating/index.js

```diff
@@ -12,7 +12,7 @@ import { ProductRating } from '@woocommerce/atomic-components/product';
 import sharedConfig from '../shared-config';
 
 const blockConfig = {
-	title: __( 'Product Rating', 'woo-gutenberg-products-block' ),
+	title: __( 'Average Rating', 'woo-gutenberg-products-block' ),
```

Comment:
I wonder if this is too vague.  Average rating of what? Some alternatives:

- `Product Rating (avg)`
- `Average Product Rating`
- `Product Average Rating`
- `Product Rating Average`
- leave unchanged?

Reply:
Maybe check to see how well this is translated to help decide if it's okay to break any existing translations?